edgelet/docker-rs/src/lib.rs

```diff
@@ -8,7 +8,7 @@
     unused_imports,
     unused_mut
 )]
-#![cfg_attr(feature = "cargo-clippy", allow(clippy, clippy_pedantic))]
+#![cfg_attr(feature = "cargo-clippy", allow(clippy::all, clippy::pedantic))]
```

Comment:
With scoped tool lints you don't need the `cfg_attr` condition any more. So this can be just `#![allow(clippy::all, clippy::pedantic)]`, and same for the others.